lib/rails-jquery-autocomplete/autocomplete.rb

```diff
@@ -49,30 +49,36 @@ def autocomplete(object, method, options = {}, &block)
             'active_record'
           end
         end
-        define_method("get_autocomplete_order") do |method, options, model=nil|
-          method("#{get_prefix(get_object(options[:class_name] || object))}_get_autocomplete_order").call(method, options, model)
+        define_method("get_autocomplete_order") do |object, method, options, model=nil|
```

Comment:
Shadowing outer local variable - `options`.<br>Line is too long. [87/80]<br>Surrounding space missing in default value assignment.